File: BackEnd/src/main/java/com/unosquare/admin_core/back_end/entity/Team.java
package com.unosquare.admin_core.back_end.entity;

import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.*;

@NoArgsConstructor
@Entity
@Data
@Table(name = "Team")
public class Team implements java.io.Serializable {

    @Id
    @SequenceGenerator(name="seq",sequenceName="team_team_id_seq", allocationSize = 1)
    @GeneratedValue(strategy=GenerationType.SEQUENCE, generator="seq")
    @Column(name = "team_id", unique = true, nullable = false)
    private int teamId;

    @ManyToOne
    @JoinColumn(name = "client_id")
    private Client client;

    private String teamName;

    private String contactName;

    private String contactEmail;

    public Team(int teamId){
        this.teamId = teamId;
    }
}

File: v2/AdminCore.DAL/Models/Team.cs
using System.Collections.Generic;
using AdminCore.DAL.Models.Base;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace AdminCore.DAL.Models
{
  [Table("team")]
  public class Team : ISoftDeletable
  {
    [Key]
    [Column("team_id")]
    public int TeamId { get; set; }

    [Column("client_id")]
    public int ClientId { get; set; }

    [Column("team_name")]
    public string TeamName { get; set; }

    [Column("contact_email")]
    public string ContactEmail { get; set; }

    [Column("contact_name")]
    public string ContactName { get; set; }

    [ForeignKey("ClientId")]
    public virtual Client Client { get; set; }

    public virtual ICollection<Contract> Contracts { get; set; }
  }
}